presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -359,6 +359,8 @@ public void testGrouping()
             throws Exception
     {
         analyze("SELECT a, b, sum(c), grouping(a, b) FROM t1 GROUP BY GROUPING SETS ((a), (a, b))");
+        analyze("SELECT grouping(t1.a) FROM t1 GROUP BY a");
+        analyze("SELECT grouping(b) FROM t1 GROUP BY t1.b");
```

Comment:
:+1:

you could also have a negative test (negative, because GROUPING doesn't work with expressions)

```
SELECT grouping(a.someField)
FROM (VALUES ROW(CAST(ROW(1) AS ROW(someField BIGINT)))) t(a)
GROUP BY a.someField;
```